### PR TITLE
chore(tabs): Minimal UI rinse

### PR DESCRIPTION
**Pull request by fk**
A bunch of minimal changes that TL;DR make the `selected` state a bit brighter and simpler, and put the bottom border of the tabs container behind the selected state of the tab items.  
Please see commit messages for details.

Here's a before/after GIF – this PR's version is 2nd and has `Sites`, `Members`, `Settings` as tab item titles:

tabs

In Gatsby Cloud (before on top, this PR at bottom):

image

### CODE REVIEW

**Comment by fk** [comment]
(removing the request for Greg as he might be OOO today – maybe someone else can take a look)

**Review by jxnblk — APPROVED** [review]

File: src/components/Tabs/Tabs.styles.tsx [hunk 1]
```diff
@@ -73,12 +74,11 @@ const tabButtonCss: ThemeCss = theme => ({
...
 
   "&:after": {
-    borderTopLeftRadius: theme.radii[1],
-    borderTopRightRadius: theme.radii[1],
+    borderRadius: theme.radii[1],
     bottom: 0,
     content: `""`,
     left: 0,
-    height: `4px`,
+    height: `2.5px`,
```
Comment (greglobinski) [inline src/components/Tabs/Tabs.styles.tsx:81]: Does it work? I mean the half pixel value?

File: src/components/Tabs/Tabs.styles.tsx [hunk 2]
```diff
@@ -88,17 +88,17 @@ const tabButtonCss: ThemeCss = theme => ({
...
 
 const tabButtonSelectedCss: ThemeCss = theme => ({
-  fontWeight: theme.fontWeights.bold,
+  fontWeight: theme.fontWeights.semiBold,
   color: theme.colors.grey[70],
   "&:after": {
-    background: theme.colors.purple[60],
-    transform: `translateY(40%)`,
+    background: theme.colors.purple[40],
+    transform: `translateY(0%)`,
```
Comment (greglobinski) [inline src/components/Tabs/Tabs.styles.tsx:101]:
```suggestion
    transform: `translateY(0)`,
```

File: src/components/Tabs/Tabs.styles.tsx [hunk 3]
```diff
@@ -88,17 +88,17 @@ const tabButtonCss: ThemeCss = theme => ({
 
   "&:hover": {
     "&:after": {
-      transform: `translateY(40%)`,
+      transform: `translateY(0%)`,
```
Comment (greglobinski) [inline src/components/Tabs/Tabs.styles.tsx:91]:
```suggestion
      transform: `translateY(0)`,
```

File: src/components/Tabs/Tabs.styles.tsx (hunk 1 above)
Comment (fk) [reply src/components/Tabs/Tabs.styles.tsx:81]: Ah don't get notifs for this repo, sorry.
Yes, but only tested on a Hi-DPI screen :/ – we could also do `0.15625rem` ;) … or `3px`.

Comment (aghreed) [reply src/components/Tabs/Tabs.styles.tsx:81]: Hmmm sounds like browsers handle it differently: https://stackoverflow.com/questions/9080633/can-a-css-pixel-be-a-fraction

**Comment by fk** [comment]
Let's give it a go! :v:

**Pull request merged**